### PR TITLE
Baby profile - first draft

### PR DESCRIPTION
**Pull request by Netanel-f**
merge me please :)

### CODE REVIEW

File: app/src/main/java/com/ux/ok_baby/BabyProfileActivity.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            if (resultCode == RESULT_OK) {
+                if (data != null) {
+                    String filePath = data.getData().getPath();
+                    android.util.Log.d(TAG, "onActivityResult: filepath is " + filePath);
+                    final Uri uri = data.getData();
+                    Log.d(TAG, "Uri: " + uri.toString());
+                    /* TODO:
+                           The next code is needed if we want immediate upload of img to firebase
+                    */
+//                    viewModel.uploadProfileImageToStorage(MainActivity.getCurrentUserID(), uri).observe(getViewLifecycleOwner(), new Observer<String>() {
```
Comment (sapirely) [inline app/src/main/java/com/ux/ok_baby/BabyProfileActivity.java]: not exactly what we want

File: app/src/main/java/com/ux/ok_baby/BabyProfileActivity.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    }
+
+    private void setupUpdateButton() {
+        update_profile_btn.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                /* TODO
+                    check whether a profile picture had been chosen
+                 */
+                if (!setBabyName()) {
```
Comment (sapirely) [inline app/src/main/java/com/ux/ok_baby/BabyProfileActivity.java:75]: maybe listener instead?

File: app/src/main/java/com/ux/ok_baby/BabyProfileActivity.java [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    private void loadFromFirebase() {
+        /* TODO:
+            load existing image from firebase
+            load existing name
+            load existing d.o.b
+         */
+    }
+
+    private boolean setBabyName() {
+        String baby_name_String = baby_name.getText().toString();
```
Comment (sapirely) [inline app/src/main/java/com/ux/ok_baby/BabyProfileActivity.java]: lower case names for variables

**Review by sapirely — CHANGES_REQUESTED** [review]
Nice work!

File: app/src/main/java/com/ux/ok_baby/BabyProfileActivity.java (hunk 1 above)
Comment (Netanel-f) [reply app/src/main/java/com/ux/ok_baby/BabyProfileActivity.java]: got it.

File: app/src/main/java/com/ux/ok_baby/BabyProfileActivity.java (hunk 3 above)
Comment (Netanel-f) [reply app/src/main/java/com/ux/ok_baby/BabyProfileActivity.java]: fixed.

File: app/src/main/java/com/ux/ok_baby/BabyProfileActivity.java [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    private void loadFromFirebase() {
+        /* TODO:
+            load existing image from firebase
+            load existing name
+            load existing d.o.b
+         */
+    }
+
+    private boolean setBabyName() {
+        String baby_name_string = baby_name.getText().toString();
```
Comment (sapirely) [inline app/src/main/java/com/ux/ok_baby/BabyProfileActivity.java]: maybe change to camel case

**Review by sapirely — CHANGES_REQUESTED** [review]
good job!

**Review by sapirely — DISMISSED** [review]
great!

**Review by sapirely — APPROVED** [review]
good

**Pull request merged**